Plugins/Public/base_plugin/CoreModule.cpp

```diff
@@ -576,23 +576,14 @@ float CoreModule::SpaceObjDamaged(uint space_obj, uint attacking_space_obj, floa
 {
 	base->SpaceObjDamaged(space_obj, attacking_space_obj, curr_hitpoints, new_hitpoints);
 
-	if (set_holiday_mode || (base->basetype == "jumpgate") || (base->basetype == "jumphole") || (base->basetype == "airlock") || (base->basetype == "planet"))
```

Comment:
don't we still need those basetype checks? or are we expecting to get away with it because they should have base->invulnerable = 1?

Reply:
Those basetypes should be always marked as logic = 0, invulnerable = 1, making those checks completely superfluous.